src/coreclr/tools/Common/TypeSystem/Common/Utilities/CustomAttributeTypeNameParser.cs

```diff
@@ -52,8 +57,33 @@ public static class CustomAttributeTypeNameParser
 
             TypeDesc type = resolver.Resolve(parsed);
 
-            typeWasNotFoundInAssemblyNorBaseLibrary = resolver._typeWasNotFoundInAssemblyNorBaseLibrary;
             return type;
+
+            static bool IsFullyQualified (TypeName typeName)
```

Comment:
We don't do this around here

```suggestion
            static bool IsFullyQualified(TypeName typeName)
```